.semaphore/semaphore.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022- The FIAAS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+version: v1.0
+name: fiaas.github.io CI build
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu2004
```

Comment:
2004 sounds old! :-o

Reply:
It looks like Semaphore mainly has Ubuntu LTS images, and 22.04 isn't available yet